src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -1000,6 +1001,25 @@ def __delitem__(self, key: str) -> None:
         """
         self._del_item(key)
 
+    @staticmethod
+    def _get_type_and_name(type_and_name):
+        return type_and_name.split(":", maxsplit=1)
+
+    def _compare_type(self, obj_type):
+        child_obj_type = self.path[-1][0]
+        return child_obj_type == obj_type
+
+    def get_state(self):
+        parent_state = PyMenu(self.service, self.rules, self.path[:-1]).get_state()
+        returned_state = {}
+
+        for key, value in parent_state.items():
+            type_and_name = self._get_type_and_name(key)
+            if len(type_and_name) == 2 and self._compare_type(type_and_name[0]):
+                returned_state[type_and_name[1]] = value
+
+        return OrderedDict(sorted(returned_state.items()))
```

Comment:
Is `OrderedDict` required?
```
>>> for k,v in dict(sorted(dict(z=5,y=2,x=3,a=1,c=3,b=4).items())).items():
...   print(k, v)
...
a 1
b 4
c 3
x 3
y 2
z 5
```

Reply:
right @seanpearsonuk, Thank you. I have removed the OrderedDict implementation.